bottles/frontend/ui/dialog-gamescope.blp

```diff
@@ -179,6 +179,36 @@ template GamescopeDialog : .AdwWindow {
           }
         }
       }
+
+      .AdwPreferencesGroup {
+        title: _("FSR");
+
+        .AdwActionRow {
+          title: _("Enable");
+          activatable-widget: "switch_fsr";
+
+          Switch switch_fsr {
+            valign: center;
+          }
+        }
+
+        .AdwActionRow {
+          title: _("Sharpening Strength");
+
+          SpinButton {
+            numeric: true;
+            valign: center;
+            adjustment:
+            Adjustment spin_sharpening_strength {
+              step-increment: 4;
+              upper: 20;
+            }
+
+            ;
+          }
+        }
+      }
+
```

Comment:
In my opinion, it makes more sense to move these above "Miscellaneous", since it's normal to put "Others" sections in the end.

I'm not too fond of the interface. I'm experimenting locally, but I'll see if we could come up with something better.

Reply:
Yeah, I can't think of a better design that fits with the current dialog. In this case, can you move the FSR section above Miscellaneous? I'll just merge it afterwards.